packages/tasit-act/src/tasitAct.test.js

```diff
@@ -0,0 +1,83 @@
+import Contract from "./tasitAct";
+import { expect, assert } from "chai";
+// Note: This file is originally genarated by `tasit-contracts` and was pasted here manually
+// See https://github.com/tasitlabs/TasitSDK/issues/45
+import { abi as contractABI } from "./helpers/SimpleStorage.json";
+// Note: Under the current `tasit-contracts` setup SimpleStorage aways will deployed with this address
+// See https://github.com/tasitlabs/TasitSDK/pull/59#discussion_r242258739
+const contractAddress = "0x6C4A015797DDDd87866451914eCe1e8b19261931";
+
+describe.skip("Contract", function() {
+  let simpleStorage;
+
+  beforeEach("should connect to and existing contract", async () => {
+    // Connection with blockchain
+    // Ethers.js allows sets a "connector" (provider or already connected wallet) on constructor of Contract
+    // or later with contract.connect() method.
+    // How connection should be made in here?
+    // - Make sense always uses `tasit-account` to create connected account instead of interact with provider object?
+    // - If Account encapsulates the provider object, Contract will connect with blockchain thru Account.
+    // I think that the point here is about how/if we will expose Provider object
+    simpleStorage = new Contract(contractAddress, contractABI);
```

Comment:
```suggestion
    const simpleStorage = new Contract(contractAddress, contractABI);
```

Reply:
@marcelomorgado Maybe this change isn't warranted if simpleStorage is set up in the beforeEach to be used in multiple tests

Reply:
Yes. Same as before.